### PR TITLE
Code Quality: Formatting Issue with Dimensions and Synchronization Labels

### LINKED ISSUE #15610: Bug: Formatting Issue with Dimensions and Synchronization Labels
### Description

I have noticed a formatting issue with the labels "dimensions" and "synchronization" when hovering over an image. The colon is directly attached to the word "dimensions" resulting in "dimensions:" instead of "dimensions :". The same problem occurs with the "synchronization" line. To fix this, the colons should be removed from the following lines of code in the file "src/Files.App/Data/Items/ListedItem.cs" on line 49-50

```
if (!string.IsNullOrWhiteSpace(DimensionsDisplay))
    tooltipBuilder.Append($"{Environment.NewLine}{"PropertyDimensions".GetLocalizedResource()}: {DimensionsDisplay}");
if (SyncStatusUI.LoadSyncStatus)
    tooltipBuilder.Append($"{Environment.NewLine}{"syncStatusColumn/Header".GetLocalizedResource()}: {syncStatusUI.SyncStatusString}");
```

and ensure that "dimensions" and "dimensions:", as well as "status" and "status:", are properly localized.

### Steps To Reproduce

1. Hover over an image file
2. Hover over a file stored in the cloud

### Requirements

Correct the code so that the labels are formatted properly, with a space between the word and the colon.

### Files Version

3.4.14.0

### Windows Version

10.0.22631.3737

### Log File

Capture d'écran 2024-06-17 125651

### PR DESCRIPTION
**Pull request by Lukiluc29**
**Resolved / Related Issues**

To prevent extra work, all changes to the Files codebase must link to an approved issue marked as `Ready to build`. Please insert the issue number following the hashtag with the issue number that this Pull Request resolves.
- Closes #15610 

**Steps used to test these changes**

Stability is a top priority for Files and all changes are required to go through testing before being merged into the repo. Please include a list of steps that you used to test this PR.

1. Opened Files
2. Hover over an image file
3. Hover over a file stored in the cloud

### CODE REVIEW

File: src/Files.App/Strings/en-US/Resources.resw [hunk 1]
```diff
@@ -3853,4 +3853,10 @@
   <data name="EditInNotepadDescription" xml:space="preserve">
     <value>Edit the selected file in Notepad</value>
   </data>
+  <data name="PropertyDimensionsColon" xml:space="preserve">
+    <value>Dimensions:</value>
+  </data>
+  <data name="syncStatusColumn/HeaderColon" xml:space="preserve">
```
Comment (0x5bfa) [inline src/Files.App/Strings/en-US/Resources.resw:3859]: Lets make it `StatusWithColon`

also syncStatusColumn/Header can be just `Status` (this is a good time to remove slash and make it PascalCase).

File: src/Files.App/Data/Items/ListedItem.cs [hunk 2]
```diff
@@ -47,9 +47,9 @@ public string ItemTooltipText
 				if (!string.IsNullOrWhiteSpace(FileSize))
 					tooltipBuilder.Append($"{Environment.NewLine}{"SizeLabel".GetLocalizedResource()} {FileSize}");
 				if (!string.IsNullOrWhiteSpace(DimensionsDisplay))
-					tooltipBuilder.Append($"{Environment.NewLine}{"PropertyDimensions".GetLocalizedResource()}: {DimensionsDisplay}");
+					tooltipBuilder.Append($"{Environment.NewLine}{"PropertyDimensionsColon".GetLocalizedResource()} {DimensionsDisplay}");
 				if (SyncStatusUI.LoadSyncStatus)
-					tooltipBuilder.Append($"{Environment.NewLine}{"syncStatusColumn/Header".GetLocalizedResource()}: {syncStatusUI.SyncStatusString}");
+					tooltipBuilder.Append($"{Environment.NewLine}{"syncStatusColumn/HeaderColon".GetLocalizedResource()} {syncStatusUI.SyncStatusString}");
```
Comment (0x5bfa) [inline src/Files.App/Data/Items/ListedItem.cs:52]:
```suggestion
					tooltipBuilder.Append($"{Environment.NewLine}{"StatusWithColon".GetLocalizedResource()} {syncStatusUI.SyncStatusString}");
```

File: src/Files.App/Strings/en-US/Resources.resw [hunk 3]
```diff
@@ -512,7 +512,7 @@
   <data name="TabStripDragAndDropUIOverrideCaption" xml:space="preserve">
     <value>Move tab here</value>
   </data>
-  <data name="syncStatusColumn.Header" xml:space="preserve">
+  <data name="Status" xml:space="preserve">
     <value>Status</value>
   </data>
```
Comment (0x5bfa) [inline src/Files.App/Strings/en-US/Resources.resw:517]: Found that this isn't used anymore
```suggestion
```

**Review by 0x5bfa — COMMENTED** [review]
Will cause build error

File: src/Files.App/Data/Items/ListedItem.cs (hunk 2 above)
Comment (Lukiluc29) [reply src/Files.App/Data/Items/ListedItem.cs]: Thank you very much, I had forgotten to change that.

File: src/Files.App/Strings/en-US/Resources.resw (hunk 3 above)
Comment (Lukiluc29) [reply src/Files.App/Strings/en-US/Resources.resw]: I was considering removing it because I couldn't find anything related to it.

Comment (Lukiluc29) [reply src/Files.App/Strings/en-US/Resources.resw]: I just saw that it's for the status column in the detailed layout.

Comment (0x5bfa) [reply src/Files.App/Strings/en-US/Resources.resw]: Ah sorry i missed, lets revert this change.

Comment (Lukiluc29) [reply src/Files.App/Strings/en-US/Resources.resw]: Even I didn't see it.

**Comment by Lukiluc29** [comment]
Now I think it's good.

**Review by 0x5bfa — APPROVED** [review]

**Review by yaira2 — APPROVED** [review]

**Comment by yaira2** [comment]
I'm marking this as "Code Quality" because it fixes a bug that wasn't released to stable. This helps me tell the bug fixes apart when preparing the changelog.

**Pull request merged**